src/main/webapp/i18n/de/featureOverview.json

```
@@ -1,9 +1,9 @@
 {
     "featureOverview": {
-        "title": "Online Klausuren in Artemis",
-        "featureDetails": "Feature Details",
+        "title": "Onlineklausuren in Artemis",
+        "featureDetails": "Feature-Details",
         "instructor": {
-            "pageTitle": "Klausur Features für Instructors",
+            "pageTitle": "Klausurfeatures für Instructors",
```

I guess we could use this:
```suggestion
            "pageTitle": "Klausurfeatures für Instruktoren",
```